augur/cli/run.py

```diff
@@ -6,6 +6,8 @@
 from copy import deepcopy
```

Comment:
The `augur run` command got moved to `augur backend start`, in a new `augur/cli/backend.py` file. The structure of the new file is almost identical to the old one; I think all you'll need to do is copy the relevant code over to the `start()` method defined in `backend.py`.